goals.md

```diff
@@ -9,29 +9,29 @@ Based on this, we would like to bring the goals to the TODO Group for review and
 
 The goals will be revisited and updated yearly.
 
-## 2022
+## [PROPOSED] 2022
 
-### 1. Cultivate growth and adoption of OSPOs
+### 1 Cultivate growth and adoption of OSPOs
 
-* Release at least 6 new OSPO case studies
-* Launch 6 new OSPO guides
-* Host 12 OSPOlogy sessions
-* Build out and run OSPO Survey 2022
-* Create and publish a OSPO Maturity Model
+- Release at least 6 new OSPO case studies
+- Launch 6 new OSPO guides
+- Host 12 OSPOlogy sessions
+- Build out and run OSPO Survey 2022
+- Create and publish a OSPO Maturity Model
 
-## 2. Grow the membership base
+## 2 Grow the membership base
 
-* Grow TODO membership by 20%
-* Cultivate relationships with other OSPO adjacent communities like CHAOSS
-* Grow OSPO Ambassadors by 
+- Grow TODO membership by 20%
+- Cultivate relationships with other OSPO adjacent communities like CHAOSS
+- Grow OSPO Ambassadors by 
 
-## 3. Create value for all members
+## 3 Create value for all members
 
-* Run at least 2 surveys to get feedback, >80% satisfaction
-* Improve and refresh TODO Website, increase traffic by 20%
-* Host at least 2 OSPOCons
-* Host first OSPOCon in APAC
-* Break down silos and enhance communication across channels (slack, mailing lists, Social media)
+- Run at least 2 surveys to get feedback, >80% satisfaction
+- Improve and refresh TODO Website, increase traffic by 20%
+- Host at least 2 OSPOCons
+- Host first OSPOCon in APAC
+- Break down silos and enhance communication across channels (slack, mailing lists, Social media)
```

Comment:
This is vague and difficult to action. What is the desired outcome here? What does success look like?

Reply:
This topic refers to two kinds of silos that TODO has:

1) **Silos related with geographical location:** Improve communication and reach out to TODO Community and contributors across different regions ( e.g not only having community meetings in an AMER or EMEA friendly timezone, but also in APAC; have multiple Touchpoint meetings in different hours or being organized by local moderators)

2) **Silos related to organization's restrictions:** For instance, some folks in the community can't access to TODO slack channel because of internal platform restrictions. That's why implementing actions that allow all the community to stay up to date about OSPO Discussions, TODO news, resources, and where to search for contribution guidelines are needed. This might include, improving website documentation, promoting OSPONews, posting frequent Community news and reminders in the TODO Mailing list, replicating slack discussions at OSPO Discussions GH channel, and more.